index.js

```diff
@@ -16,9 +16,19 @@ function promisifyOwnProperties(obj) {
 }
 
 module.exports = obj => {
+  if (!obj) {
+    return obj;
+  }
+
+  const typeOfObj = typeof obj;
```

Comment:
not a big deal, but it's actually faster to repeat the `typeof obj === 'foo'` every time it's used, because engines can optimize it - whereas this, they can't.